code/modules/clothing/glasses/_glasses.dm

```diff
@@ -109,7 +109,6 @@
 	icon_state = "night"
 	item_state = "glasses"
 	darkness_view = 8
-	vision_flags = SEE_BLACKNESS

```

Comment:
#31114

Reply:
WELP

Reply:
I've tested this on both 511 and 512 without any issues, although I'm looking into it at the moment

Reply:
so after going through many older versions of the code, I've figured out it was probably something to do with parallax.
I haven't exactly found what commit started space-vision, nor what commit fixed it, but it's definitely not a problem anymore.
going to push another commit for nvg eyes and then go to bed will post pics of proof this works fine tmrw if requested

Reply:
please post pictures, i'm a little hesitant to essentially revert antur's fixes

Reply:
This is *probably* fine the real fix was moving the game plane from 0.